src/vma/util/sys_vars.cpp

```diff
@@ -955,8 +967,9 @@ void mce_sys_var::get_env_params()
 	tscval_t tsc_per_second = get_tsc_rate_per_second();
 	rx_delta_tsc_between_cq_polls = tsc_per_second * rx_cq_drain_rate_nsec / NSEC_PER_SEC;
 
-	if ((env_ptr = getenv(SYS_VAR_GRO_STREAMS_MAX)) != NULL)
+	if ((env_ptr = getenv(SYS_VAR_GRO_STREAMS_MAX)) != NULL) {
```

Comment:
all if cases have no brackets. why do you want to add for those cases ?

Reply:
I think it was a result of searching a way to process socketxtreme limitations. In any case I do not any harm using brackets.